Makefile

```diff
@@ -79,6 +79,10 @@ test-e2e:
 test-demo:
 	go test -v github.com/redhat-developer/odo/tests/e2e --ginkgo.focus="katacodaDemo" -ginkgo.v
 
+.PHONY: test-update-e2e
+test-update-e2e:
```

Comment:
I suspect that those tests are never run on travis :-(

Reply:
Yep, need to make a travis.yml entry for this. Something like this: https://github.com/redhat-developer/odo/blob/master/.travis.yml#L46

Reply:
@kadel @ashetty1 Done, it's running separately as a separate job now

Reply:
@mik-dass Fantastic!